apps/meteor/server/services/omnichannel/logger.ts

```diff
@@ -0,0 +1,3 @@
+import { Logger } from '../../lib/logger/Logger';
+
+export const queueLogger = new Logger('OmnichannelQueue');
```

Comment:
Could you cleanup this old logger? I don't think anyone is using it and since they both have the same name, it might create confusion

Reply:
This is the new logger lol

But i removed the old one :awesome: